src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String REPORT_HEADER = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String generateReport(List<Fruit> fruitList) {
```

Comment:
Add Storage as class field
```suggestion
    public String generateReport() {
```